index.bs

```diff
@@ -2294,6 +2409,15 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
     : [=randomized response output configuration/trigger specs=]
     :: |triggerSpecs|
 
+1. Let |attributionScopeLimit| be the result of running
```

Comment:
The attribution scopes will be stored in the cache, should we impose a max limit?

Reply:
Why would they be different than other lists where we stipulated favouring header size over individual limits?

Reply:
We only remove the size limit for trigger registrations if they're not stored in the cache. For source registrations we still have the limits, for example, for filters.

Reply:
Ah, oops, good point!

Reply:
Good point. Added a max limit with 20 for this.